### ISSUE TITLE
Board editor: connections only show up after a wire placement is initiated in schematic editor

### ISSUE THREAD

**Issue by kotchin** [LibrePCB/LibrePCB#1171]
Grützi, thank you for creating LibrePCB, I enjoy using it :)

This is a minor bug, which I expect the vast majority of people to not be exposed to. It also has very few consequences. It was just the result of my evaluation to see whether I understood how LibrePCB worked and if I created my device properly and was able to see the internal connections having been placed internally.

##### VERSION / OS / ENVIRONMENT
```
LibrePCB Version: 0.1.7
Git Revision: ba553c6
Qt Version: 5.15.9 (built against 5.15.6)
Operating System: Fedora (KDE spin)
```

##### SUMMARY
When a package has multiple pads internally connected to the same pin (e.g. GND), if this component is placed alone in the schematic editor, without making any wiring connections whatsoever and then viewed in the board editor, the pads of the component which should be interconnected (as they are part of the same pin) don't appear interconnected, until a wire placement in the schematic editor is initiated (doesn't even need to be fully placed, just initiated).

##### STEPS TO REPRODUCE
1) Create a device, where a package has multiple pads belonging to the same pin from the symbol in the schematic editor (e.g. the symbol has one GND pin, which corresponds to 3 GND pads in the package).
2) place this device alone, in the schematic editor, without placing anything else
3) when placing the corresponding package in the board editor, no pads are shown to require to be connected
4) start placing a wire in the schematic editor where the symbol was previously placed in 2
5) witness the connections suddenly appearing the board editor

##### EXPECTED RESULTS
The results of (5) should appear without needing (4).

##### ACTUAL RESULTS
step 4 was required before results of step 5 could be observed.

##### ADDITIONAL INFORMATION
Here you can witness the missing connections (after step 3)
![librepcb_1](https://github.com/LibrePCB/LibrePCB/assets/6894486/7363b869-9903-4da3-885a-e944aebc8cea)
Here you can witness the connections having appeared (after step 5)
![librepcb_2](https://github.com/LibrePCB/LibrePCB/assets/6894486/fbc24744-c07c-4ec4-9865-9036180cb9bf)

**Comment by ubruhin** [comment]
Hi, thanks for reporting! This behavior is indeed not ideal but I consider it as low priority as it only affects pins not connected in schematics ;)
